README.md

```diff
@@ -1,7 +1,12 @@
 # Staged Updates on balena
 
 This repo is a collection of scripts to demonstrate some of the new API endpoints offered on balenaCloud to enable the more fine grained control of app updates across a fleet of devices.
-These scripts simply show how to use the available primitives and in the near future this functionality will be surfaced on the UI and via the CLI.
+
+> Update: Since this repo was created, the release pinning functionality was made available through
+> the web dashboard, and masterclass documentation was added on using the balena API (e.g. `curl`
+> commands) to achieve the same. References:  
+> https://www.balena.io/docs/learn/deploy/release-strategy/release-policy/  
+> https://www.balena.io/docs/learn/more/masterclasses/fleet-management/#6-release-policy
```

Comment:
I thought of adding this note here because `balena help preload` (help for the balena CLI `preload` command) points to this repo / Readme. I'll be updating the CLI to point to the masterclass instead (probably), but older CLI versions would still point to this repo.